qiskit/qpy/type_keys.py

```diff
@@ -322,6 +329,8 @@ class ScheduleOperand(TypeKeyBase):
     # Discriminator and Acquire instance are not serialzied.
     # Data format of these object is somewhat opaque and not defiend well.
     # It's rarely used in the Qiskit experiements. Of course these can be added later.
+    KERNEL = b"k"
```

Comment:
We should remove the preceding comment.

Reply:
Done! :) 👉 https://github.com/Qiskit/qiskit-terra/pull/10327/commits/ed7515dd5aff98120824998f3476d1a271c7dbbd